### ISSUE TITLE
Fix DeprecationWarning from XMLParser.html

### ISSUE THREAD

**Issue by turicas** [tiran/defusedxml#37]
I'm using [openpyxl](https://bitbucket.org/openpyxl/openpyxl) inside a library and got a lot of `DeprecationWarning`s from defusedxml ([openpyxl uses defusedxml internally](https://bitbucket.org/openpyxl/openpyxl/pull-requests/300/use-defusedxml-to-guard-against-various/)), like this one:

```
defusedxml/ElementTree.py:68: DeprecationWarning: The html argument of XMLParser() is deprecated
    _XMLParser.__init__(self, html, target, encoding)
```

defusedxml version is 0.5.0.

**Comment by tiran** [comment]
Please try 0.6.0rc1

**Comment by tiran** [comment]
I released 0.6.0. You should no longer get a deprecation warning for html argument.

**Comment by turicas** [comment]
Thanks! :)
